bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/MeasureCapability.java

```diff
@@ -90,4 +83,13 @@ private void updateMeasures(WeatherApi api, String deviceId, @Nullable String mo
                     }
                 });
     }
+
+    @Override
+    protected List<NAObject> getFreshData(WeatherApi api) {
+        String bridgeId = handler.getBridgeId();
+        String deviceId = bridgeId != null ? bridgeId : handler.getId();
+        String moduleId = bridgeId != null ? handler.getId() : null;
+        updateMeasures(api, deviceId, moduleId);
+        return List.of();
```

Comment:
As the result is an empty list, I understand cache will not be used.
Is it expected ?

Reply:
You're correct. Let me rethink this.

Reply:
I have modified the way CacheWeatherCapability behaves to take in account potential empty list result.